app/assets/stylesheets/app/session_preview.scss

```diff
@@ -0,0 +1,3 @@
+.session-preview-element__highlighted {
```

Comment:
I would just call it highlight.

Also, what IS highlight? where did the need come from?